### ISSUE TITLE
%spaceAltitudeThreshold = 2000000 for most bodies

### ISSUE THREAD

**Issue by Paul-kingtiger** [KSP-64k/64k#12]
I noised the spaceAltitudeThreshold is set for 200000 for most of the bodies.  I thought this was the altitude that contracts use to know when you're in space.  I.e how high you need to be to complete that sort of contract.

I'm assuming these need to be set to custom for each body.  With the value being above the atmosphere for planets with one.  No idea what the values should be for planets without an atmosphere?

**Comment by Felger** [comment]
spaceAltitudeThreshold is the science differentiation point between Flying In Space High and Flying In Space Low.

**Comment by Paul-kingtiger** [comment]
I've pretty much fixed the water level issue, which has also resolved the z fighting around the coastline.  I'm creating new scaled space assets now. 

I've also reset the spaceAltitudeThreshold, so that it is the same as the body radius.  That seemed a good starting point, but we can always customize each body for better values.

I think we'll be ready to release tomorrow.

**Comment by Felger** [comment]
Awesome stuff! I will be out of town this weekend, but I believe I'll have a little time tomorrow afternoon to check over anything we need prior to release.

**Comment by Felger** [comment]
It also occurs to me, we might want to update to the most recent Kopernicus release in the repo prior to publishing.

**Comment by Paul-kingtiger** [comment]
Agreed. I've actually been using 0.9 for testing so this should be easy enough. I'm going through the scaled space atmosphere colors and specular values at the moment, we're definitely at the polish stage.

Sent from my Windows Phone
________________________________
